src/app/shared/doc-viewer/doc-viewer.ts

```diff
@@ -107,12 +116,21 @@ export class DocViewer implements OnDestroy {
 
     Array.prototype.slice.call(exampleElements).forEach((element: Element) => {
       const example = element.getAttribute(componentName);
+      // const file = element.getAttribute(file);
+      // this.lines = element.getAttribute(lines);
+      // temporary data to test flow of info
+      this.lines = [0, 5];
+      const file = 'HTML';
       const portalHost = new DomPortalOutlet(
           element, this._componentFactoryResolver, this._appRef, this._injector);
       const examplePortal = new ComponentPortal(componentClass, this._viewContainerRef);
       const exampleViewer = portalHost.attach(examplePortal);
       if (example !== null) {
         (exampleViewer.instance as ExampleViewer).example = example;
```

Comment:
extract to variable?

Reply:
Did you mean to resolve this? I would extract a variable so you don't have to repeat `as ExampleViewer` several times.